examples-generated/instance/v1alpha1/config.yaml

```diff
@@ -9,11 +9,9 @@ metadata:
 spec:
   forProvider:
     booted: true
-    devices:
-    - sda:
-      - diskIdSelector:
-          matchLabels:
-            testing.upbound.io/example-name: boot
+    device:
```

Comment:
is this just from schema changes or did we have to disable some cross resource references?

Reply:
just from schema changes